lib/orwell/github.ex

```diff
@@ -58,6 +49,20 @@ defmodule Orwell.GitHub do
     end
   end
 
+  @spec find_post(binary, List, Config.t) :: {atom, binary}
+  defp find_post(id, posts, config) do
+    path =
+      posts
+      |> Enum.filter(&Regex.match?(~r/.*\/#{id}-[^\/]*.md/, &1["path"]))
```

Comment:
I know but they are in a different module. Also, notice the interpolation. I want to match the specific id, not to retrieve an id.